docs/products/kafka/howto/view-kafka-topic-catalog.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Manage Apache Kafka topics with topic catalog
+sidebar_label: Manage topic catalog
+limited: true
+---
+
+The Aiven for Apache Kafka® topic catalog offers a user-friendly interface to manage your Apache Kafka topics within your Aiven for Apache Kafka services.
+
+## Access topic catalog
```

Comment:
You might want to use an article before "topic catalog" coz you're using articles in the sections titles that follow.
```suggestion
## Access a/the topic catalog
```